app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ReviewsController < ApplicationController
+  before_filter :require_user
+	def create
+    #binding.pry
```

Comment:
You want to make sure you remove any unnecessary comments from the code before you merge it with a PR. Doing this while you code is fine (as is the way you used it below). But treat a PR as if it is supposed to be a polished final submission that your boss or co-workers are going to look at.